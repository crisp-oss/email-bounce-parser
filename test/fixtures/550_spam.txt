This is the mail system at host mailer.acme.email.

I'm sorry to have to inform you that your message could not
be delivered to one or more recipients. It's attached below.

For further assistance, please send mail to postmaster.

If you do so, please include this problem report. You can
delete your own text from the attached returned message.

                   The mail system

<didos@msdnports.com>: host mail.msdnports.com[170.108.9.86] said: 550 "The
    mail server detected your message as spam and has prevented delivery." (in
    reply to end of DATA command)
