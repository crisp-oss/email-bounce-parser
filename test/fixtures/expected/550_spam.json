{
  "bounce": true,
  "email": {
    "body": "This is the mail system at host mailer.acme.email.\n\nI'm sorry to have to inform you that your message could not be delivered to one or more recipients. It's attached below.\n\nFor further assistance, please send mail to postmaster.\n\nIf you do so, please include this problem report. You can delete your own text from the attached returned message.\n\nThe mail system\n\n<didos@msdnports.com>: host mail.msdnports.com[170.108.9.86] said: 550 \"The mail server detected your message as spam and has prevented delivery.\" (in reply to end of DATA command)",
    "intro": "This is the mail system at host mailer.acme.email.\n\nI'm sorry to have to inform you that your message could not be delivered to one or more recipients. It's attached below.\n\nFor further assistance, please send mail to postmaster.\n\nIf you do so, please include this problem report. You can delete your own text from the attached returned message.\n\nThe mail system",
    "error": "<didos@msdnports.com>: host mail.msdnports.com[170.108.9.86] said: 550 \"The mail server detected your message as spam and has prevented delivery.\" (in reply to end of DATA command)"
  },
  "data": {
    "error": {
      "code": {
        "basic": "550",
        "enhanced": null
      },
      "label": "The mail server detected your message as spam and has prevented delivery.",
      "type": "action_not_taken",
      "temporary": false,
      "permanent": true,
      "data": {
        "type": null,
        "blocked": false,
        "spam": true
      }
    },
    "recipient": "didos@msdnports.com",
    "server": {
      "hostname": "mail.msdnports.com",
      "ip": "170.108.9.86",
      "port": null
    },
    "command": "DATA"
  }
}